youtube-tracker/README.md

```diff
@@ -0,0 +1,51 @@
+# Track YouTube video views and comments
+
+_Tracks views, likes, and comments for provided YouTube URLs in a Google Sheet with optional email notifications._
+
+Last updated: August, 2019
+
+Track performance of public YouTube videos, including views, likes, and comments, in a Google Sheet. 
+Refresh data automatically via the YouTube Data API. Send automatic email notifications
+that summarize new YouTube video comment activity.
+
+![screenshot](https://cdn.jsdelivr.net/gh/gsuitedevs/solutions@master/youtube-tracker/screenshot.png)
+
+## Technology highlights
+- Use the [YouTube advanced service](https://developers.google.com/apps-script/advanced/youtube) to collect data on specific videos (such as number of views and comment data) via the YouTube Data API.
+- Use the [Gmail Service](https://developers.google.com/apps-script/reference/gmail/) to send notification emails when new video comments are detected. 
+- Create an [HTML template](https://developers.google.com/apps-script/guides/html/templates#calling_apps_script_functions_from_a_template) containing an Apps Script function to send emails containing HTML tables.
```

Comment:
Using the word "containing" twice in the same sentence sounds weird. Consider rewording, perhaps being less descriptive.

Reply:
How about...
Creates an [HTML template](https://developers.google.com/apps-script/guides/html/templates#calling_apps_script_functions_from_a_template) containing an Apps Script function to send custom notification emails.

Reply:
Works for me.